source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -25,8 +25,109 @@ more naturally readable.
    :manual:`Aggregation Pipeline Operators </reference/operator/aggregation/>` 
    page in the Server manual.
 
+Sample Data
+-----------
+
+To run the examples in this guide, insert documents describing locations
+and temperature readings into the ``db.tempReadings`` collection with the following
+snippet:
+
+.. code-block:: java
+   :copyable: true
+
+   MongoDatabase database = client.getDatabase("db");
+   MongoCollection<Document> myColl = database.getCollection("tempReadings");
+   
+   List<Document> docs = Arrays.asList(
+           new Document().append("location", "New York, NY").append("temp", 54),
+           new Document().append("location", "Duluth, MN").append("temp", 4),
+           new Document().append("location", "Riverside, CA").append("temp", 101),
+           new Document().append("location", "Tucson, AZ").append("temp", 92));
+
+   InsertManyResult result = myColl.insertMany(docs);
+
 Operations
 ----------
 
+Boolean Expressions
+~~~~~~~~~~~~~~~~~~~
+
+Boolean expressions evaluate arguments as booleans and return ``true`` or
+``false``. These expressions evaluate ``false``, ``null``, ``0``, and
+``undefined`` values as ``false``, and all other values as ``true``.
+
+The following table describes the boolean aggregation expression methods
+available in the {+driver-short+} and the equivalent expressions in the
+Query API:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Java Aggregation Expression
+     - Query API Expression
+
+   * - ``and()``
+     - :manual:`$and </reference/operator/aggregation/and/>`
+       
+   * - ``not()``
+     - :manual:`$not </reference/operator/aggregation/not/>`
+
+   * - ``or()``
+     - :manual:`$or </reference/operator/aggregation/or/>`
+
+   * - ``cond()``
+     - :manual:`$cond </reference/operator/aggregation/cond/>`
+
+.. example::
+   
+   The following code determines whether a temperature reading
+   is extreme depending on if the ``temp`` field is greater
+   than 90 *or* less than 10, given by the value of the
+   ``extremeTemp`` field. The output documents include only the original
+   ``location`` field and the computed ``extremeTemp`` field:
+
+   .. io-code-block::
+      :copyable: true
+   
+      .. input::
+         :language: java
+         :emphasize-lines: 5-6
+
+         var pipeline = Arrays.asList(project(
+                 fields(
+                         exclude("_id"),
+                         include("location"),
+                         computed("extremeTemp",
+                                 current().getInteger("temp").lt(of(10)).or(current().getInteger("temp").gt(of(90))))
+                 )
+         ));
```

Comment:
```suggestion
         var pipeline = Arrays.asList(project(fields(
                 exclude("_id"),
                 include("location"),
                 computed("extremeTemp",
                         current().getInteger("temp").lt(of(10))
                             .or(current().getInteger("temp").gt(of(90))))
         )));
```
The example scrolls horizontally - the above indentation should make it more readable.

You might also consider including 
```
                             .cond("extreme!", "not extreme")
```
on a new line after the "or" (with correct parens). The way the "cond" operation works (along with switch, and a few others) might be unexpected to some users, so it seems worth addressing it in some way.

Reply:
We are trying to use only one aggregation method per example, so that users can clearly see what a given method does individually.

Reply:
Tough call. Out of the dozens of expressions, there were several comments during design review mentioning that `.cond` seemed unusual. It would be nice to have two examples, but if I had to pick I would go for $cond.